_test/test_utilities/test_make_const_bin_boundaries.m

```diff
@@ -0,0 +1,64 @@
+classdef test_make_const_bin_boundaries < TestCase
+    
+    
+    properties
+    end
+    %
+    methods
+        function obj = test_make_const_bin_boundaries(varargin)
+            if nargin>0
```

Comment:
why do you support the test name overloading here rather than simply call `obj = obj@TestCase('test_make_const_bin_boundaries';);`

Reply:
The original Tests were creating test object for every tests method. We have rewritten that to have one constructor per test class. 
When the constructor was called per method, single test class name would not work. But when you run runtests, it still names test classes according to its own naming. I have never cared to investigate is it necessary indeed or one may call the classes as internal name.  Its clear, varargin is needed for a test input as a default name is provided for tests. And if you have varargin then why not use it. 

The default name is convenient when you run single tests in a pattern:
tc = test_specific_tests
tc.test_method();
then default name is needed. 

There is also minor convenience -- possibility to have the same name for different test classes. 

So, in a way, it is not really necessary any more but may be convenient and useful in special cases and the custom remain from the time it was necessary